### PR TITLE
A first past at creating a record locator API

### PR DESCRIPTION
**Pull request by khandelwal**
First: Apologies that this pull request is so long. 

This sets up Django, and adds a simple API that allows the user to request n record locators.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Django, using PostgreSQL
+Django
+psycopg2
+dj-database-url
+
+# Recordlocator
+-e git+https://github.com/khandelwal/record-locator.git#egg=recordlocator
```
Comment (seanherron) [inline requirements.txt]: Should this be forked to 18F?

File: ticketer/recordlocator/views.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+from django.shortcuts import render
+from django.http import HttpResponse
+
+from recordlocator import generator
+
+def record_locators(request):
+    """ View that generates at least one record locator, generating 'n' if
+    specified through a query parameter. """
+
+    num_locators = int(request.GET.get('n', 1))
```
Comment (seanherron) [inline ticketer/recordlocator/views.py:16]: Should we put a max on this?

**Comment by seanherron** [comment]
LGTM, put in a few inline comments to be looked at first

File: requirements.txt (hunk 1 above)
Comment (khandelwal) [inline requirements.txt]: Yes! I'll get right on that.

File: ticketer/recordlocator/views.py (hunk 2 above)
Comment (khandelwal) [inline ticketer/recordlocator/views.py:16]: You know, it's funny but I was thinking that, just as I submitted this pull request. What do you think a decent max is? I can't imagine needing more than a 100 at a time?

Comment (seanherron) [inline ticketer/recordlocator/views.py:16]: I was thinking 50, but maybe 100 is better for a large library or something.

File: ticketer/recordlocator/views.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+MAX_REQUESTABLE = 50
+
+
+def record_locators(request):
+    """ View that generates at least one record locator, generating 'n' if
+    specified through a query parameter. """
+
+    num_locators = int(request.GET.get('n', 1))
+
+    if num_locators > MAX_REQUESTABLE:
```
Comment (khandelwal) [inline ticketer/recordlocator/views.py:18]: Limit handling added here. I set it to 50.

**Pull request merged**